libraries/aws_cloudwatch_log_metric_filter.rb

```diff
@@ -0,0 +1,153 @@
+require 'aws_conn'
+
+class AwsCloudwatchLogMetricFilter < Inspec.resource(1)
+  name 'aws_cloudwatch_log_metric_filter'
+  desc 'Verifies individual Cloudwatch Log Metric Filters'
+  example <<-EOX
```

Comment:
Should we use the new ruby 2.3 `<<~EOX` see https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc